docs(menu): add toggle example

liamdebeasi opened the pull request:
This PR adds a playground example on how to use `ion-menu-toggle` with `ion-menu`.

https://ionic-docs-git-1282-toggle-ionic1.vercel.app/docs/api/menu#menu-toggle

brandyscarney commented:
Should the usage be removed from [here](https://ionic-docs-git-1282-toggle-ionic1.vercel.app/docs/api/menu-toggle#usage) then?

liamdebeasi commented:
I removed the usage in https://github.com/ionic-team/ionic-docs/pull/2548/commits/169cf3b468f7224f8bc757b3775146ee3dd2dde9 and added a link to the new menu toggle example.
